DigitalOffice/Services/UserService/Commands/LoginUserCommand.cs

```diff
@@ -0,0 +1,57 @@
+﻿using FluentValidation;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using UserService.Commands.Interfaces;
+using UserService.Database;
+using UserService.Database.Entities;
+using UserService.Models;
+using System.Security.Cryptography;
+using System.Text;
+using System.Net;
+using System.Web;
+using UserService.Repositories.Interfaces;
+
+namespace UserService.Commands
+{
+    public class LoginUserCommand : ILoginUserCommand
+    {
+        IValidator<UserCredentials> validator;
+        IUserRepository repository;
```

Comment:
Why shouldn't these fields be private and readonly?

Reply:
fixed